src/controllers/simplified-account/settings/card-payments/google-pay.controller.js

```diff
@@ -1,37 +1,90 @@
+const paths = require('@root/paths')
 const { response } = require('@utils/response')
-const { formatSimplifiedAccountPathsFor } = require('@utils/simplified-account/format')
+const { formatSimplifiedAccountPathsFor, formatValidationErrors } = require('@utils/simplified-account/format')
 const { updateAllowGooglePay } = require('@services/card-payments.service')
-const paths = require('@root/paths')
-const validateOnOffToggle = require('@utils/simplified-account/validation/on-off-toggle')
+const { validationResult } = require('express-validator')
+const { WORLDPAY } = require('@models/constants/payment-providers')
+const { updateGooglePayMerchantId } = require('@services/worldpay-details.service')
+const { googlePaySchema, GOOGLE_PAY_TOGGLE_FIELD, GOOGLE_PAY_MERCHANT_ID_FIELD } = require('@utils/simplified-account/validation/google-pay.schema')
 
+/**
+ * @param {import('@utils/types/settings/settings-request').SettingsRequest} req
```

Comment:
There's a [SimplifiedAccountRequest type](https://github.com/alphagov/pay-selfservice/blob/06201d6eedd5218c5ed4faf6198e8f5b488a462a/src/middleware/simplified-account/simplified-account-strategy.middleware.js#L89) in existence. I don't mind which one we pick but we should avoid confusion by having both.

Reply:
I agree, there's still some issues with how this type is resolved on route handlers. The IDE will complain that there is a mismatch with both of the existing types but the one I have added here does correctly extend the express request type. 

We will do some clean up on these typedefs to ensure there is no duplication of effort but I think that can be done separate to this PR